frontend/src/components/Meal/Day.js

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { DateTime } from "luxon";
+import Box from "@material-ui/core/Box";
+import Collapse from "@material-ui/core/Collapse";
+import IconButton from "@material-ui/core/IconButton";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+
+import { APP_URL } from "../../config";
+import { getBackendAuthToken } from "../../session/localStorage";
+import { useDayStyles } from "../../styles/theme";
+
+const mealsTotalKcalSum = (meals) => {
+  return meals.reduce((prev, current) => prev + current.total_kcal, 0)
+}
+
+function getMealNameAsTime(mealDatetimeStr) {
```

Comment:
Create DateTime (using luxon) from mealDatetimeStr. Depending on how you forward this string, you can use various of methods that are exposed by Luxon: https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html

Check the **Creation** paragraph.


I suggest sending epoch timestamp from Backend (integer representation of date) and using https://moment.github.io/luxon/docs/manual/parsing.html#unix-timestamps - but whatever works, if used from Luxon, works for me.